packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -129,6 +129,7 @@ class CupertinoTextSelectionToolbar extends StatelessWidget {
         delegate: TextSelectionToolbarLayoutDelegate(
           anchorAbove: anchorAbove - localAdjustment - contentPaddingAdjustment,
           anchorBelow: anchorBelow - localAdjustment + contentPaddingAdjustment,
+          fitsAbove: fitsAbove,
```

Comment:
Previously I wasn't passing this because TextSelectionToolbarLayoutDelegate could calculate it itself (see the [docs](https://master-api.flutter.dev/flutter/widgets/TextSelectionToolbarLayoutDelegate/fitsAbove.html)), but I suppose we can pass it in here to avoid calculating it twice.